src/orange/orangeinp/IntersectRegion.cc

```diff
@@ -646,6 +698,69 @@ void InfWedge::output(JsonPimpl* j) const
     to_json_pimpl(j, *this);
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Build surfaces.
+ */
+void Involute::build(IntersectSurfaceBuilder& insert_surface) const
+{
+    if (sign_ == Sign::clockwise)
+    {
+        insert_surface(Sense::outside,
+                       celeritas::Involute{
+                           {0, 0},
+                           radii_[0],
+                           eumod(displacement_angle_[0], 2 * constants::pi),
+                           sign_,
+                           t_bounds_[0],
+                           t_bounds_[1] + displacement_angle_[1]
+                               - displacement_angle_[0]});
+        insert_surface(Sense::inside,
+                       celeritas::Involute{
+                           {0, 0},
+                           radii_[0],
+                           eumod(displacement_angle_[1], 2 * constants::pi),
+                           sign_,
+                           t_bounds_[0],
+                           t_bounds_[1] + displacement_angle_[1]
+                               - displacement_angle_[0]});
+    }
+    else
+    {
+        insert_surface(Sense::inside,
+                       celeritas::Involute{
+                           {0, 0},
+                           radii_[0],
+                           eumod(displacement_angle_[0], 2 * constants::pi),
+                           sign_,
+                           t_bounds_[0],
+                           t_bounds_[1] + displacement_angle_[1]
+                               - displacement_angle_[0]});
+        insert_surface(Sense::outside,
+                       celeritas::Involute{
+                           {0, 0},
+                           radii_[0],
+                           eumod(displacement_angle_[1], 2 * constants::pi),
+                           sign_,
+                           t_bounds_[0],
+                           t_bounds_[1] + displacement_angle_[1]
+                               - displacement_angle_[0]});
+    }
```

Comment:
This can be simplified quite a bit. I think this should work:
```suggestion
    // Make an inside and outside involute
    auto sense = (sign_ == Sign::clockwise ? Sense::outside : Sense::inside);
    for (auto i : range(2))
    {
        insert_surface(sense,
                       celeritas::Involute{
                           {0, 0},
                           radii_[0],
                           eumod(displacement_angle_[i], 2 * constants::pi),
                           sign_,
                           t_bounds_[0],
                           t_bounds_[1] + displacement_angle_[1]
                               - displacement_angle_[0]});
        sense = flip_sense(sense);
    }
```